docs/proposals/validation-tools/readme.md

```diff
@@ -0,0 +1,83 @@
+# Swagger Validation Tools VS Code Extension
+
+## Objectives and High-level plan
+
+We plan on building a one-stop shop vs-code extension that will help the swagger authors get live and prompt feedback on the quality and correctness of their swaggers. We plan on reusing and building on top of the work that was done for the Autorest Linter VS code extension and integration our tools Autorest to simplify the users experience.
+

```

Comment:
It may be useful to describe the scenarios (and priorities) we're aiming to solve (or simplify), so if we want to consider other options we can evaluate them based on these; and what may be non-goals for these tools. For example: provide swagger authors validation when creating a new swagger spec from scratch, assist with editing/updating of a swagger spec, ability to run the back-end tools as part of CI, assist with swagger spec reviews, etc.